src/text/mixins.scss

```diff
@@ -8,6 +8,8 @@
 }
 
```

Comment:
There also is `caption`. Are we using it anywhere?

Reply:
It appears that caption & small have the same properties.  I'm referencing the `iui-text(small)` mixin within `classes.scss` but no mention of `iui-text(caption)`.

Reply:
So we need to remove it, no?